packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-description.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import { TCompareAccountsCard } from 'Components/props.types';
+import { getJuridisctionDescription, getMarketType } from '../../Helpers/compare-accounts-config';
+
+const CFDCompareAccountsDescription = ({ trading_platforms, is_demo }: TCompareAccountsCard) => {
+    const market_type = getMarketType(trading_platforms);
+    const market_type_shortcode = market_type.concat('_', trading_platforms.shortcode);
+    const juridisction_data = getJuridisctionDescription(market_type_shortcode);
+
+    return (
+        <div
+            className={classNames('compare-cfd-account-text-container', {
+                'compare-cfd-account-text-container--demo': is_demo,
+            })}
+        >
+            <div className='compare-cfd-account-text-container__separator'>
+                <Text as='h1' weight='bold' size='m' align='center'>
+                    {juridisction_data.leverage}
+                </Text>
+                <Text as='p' size='xxxs' align='center'>
+                    {juridisction_data.leverage_description}
+                </Text>
+            </div>
+            <div className='compare-cfd-account-text-container__separator'>
+                <Text as='h1' weight='bold' size='m' align='center'>
+                    {juridisction_data.spread}
+                </Text>
+                <Text as='p' size='xxxs' align='center'>
+                    {juridisction_data.spread_description}
+                </Text>
+            </div>
```

Comment:
```suggestion
                <SameComponents>
                    {juridisction_data.leverage_description}
                </SameComponents> 
                <SameComponents>
                    {juridisction_data.spread}
                </SameComponents> 
```
I think all of these including the ones below is the same component 👀

Reply:
Same component but real has more details in the description.